guides/doc-Managing_Hosts/topics/Host_Management_without_Goferd.adoc

```diff
@@ -40,6 +40,8 @@ endif::[]
 
 . Remove the Katello agent on content hosts:
 +
+WARNING: Until https://bugzilla.redhat.com/show_bug.cgi?id=1850994[BZ#1850994] is resolved, if your host is installed on RHV version 4.4 or lower, do not remove the `katello-agent` package because the removed dependencies corrupt the host.
++
```

Comment:
If this is also upstream documentation, I would also mention oVirt, not only RHV.

Reply:
I would also not count on bug 1850994 being resolved because if this doc is published it will be closed as duplicate of a 4.4 bug which is already closed but the issue will persist in 4.2 and 4.3.

Reply:
Good point about oVirt. What are the oVirt versions affected?

Reply:
Great, I have updated the wording to apply your suggestions.

Reply:
Strangly, the commit does not display in this PR. While it can be seen in [compare](https://github.com/theforeman/foreman-documentation/compare/master...spetrosi:BZ%231851052?expand=1). I think GitHub works strangely today, we should wait a bit probably.

Reply:
Yes, it turned out that GitHub overcame an [incident](https://www.githubstatus.com/incidents/d60fmldlvw48) this morning I have created a new PR #209 with the same commits to workaround.